poulpe-model/src/test/java/org/jtalks/poulpe/model/entity/PoulpeUserTest.java

```diff
@@ -31,4 +35,12 @@ public Group groupWithId(long id) {
         return group;
     }
 
+    @Test
+    public void theEntityFieldsShouldBeSerialized(){
+        PoulpeUser user = PoulpeUser.withId(1);
+        byte[] serialize = SerializationUtils.serialize(user);
+        PoulpeUser serializedUser = (PoulpeUser)SerializationUtils.deserialize(serialize);
+        assertEquals(user, serializedUser, "After serialization/deserialization PoulpeUser objects must be the same");
```

Comment:
By `assertEquals` you check only the fields participating in equals() method. Instead you should use `assertReflectionEquals()`.